assemblies/lib/pom.xml

```diff
@@ -627,39 +624,6 @@
         </exclusion>
       </exclusions>
     </dependency>
-    <dependency>
```

Comment:
By removing jackson from here aren't we removing it from data-integration/lib? Doesn't this break non OSGi plugins that assume that this lib is available?
@hbfernandes ^

Reply:
In the last test stack build we have `jackson-annotations-2.9.5.jar`, `jackson-databind-2.9.5.jar` and `jackson-core-2.9.5.jar` in _data-integration/lib_. We have these three declared as dependencies of **kettle-engine**. Do we also want to make these as explicit dependencies of **assemblies/lib**?
Isn't it redundant to have these as dependencies of **assemblies/lib** and **assemblies/core/lib**?
@graimundo @hbfernandes

Reply:
https://github.com/pentaho/pentaho-kettle/pull/5485#discussion_r198820920